api_gateway/roles/ambassador/tasks/main.yml

```diff
@@ -11,3 +11,25 @@
     chart_version: 6.5.9
     release_namespace: ambassador
     create_namespace: true
+
+- name: Install service monitor to expose ambassador metrics
+  community.kubernetes.k8s:
+    state: present
+    definition:
+      apiVersion: monitoring.coreos.com/v1
```

Comment:
I think this adds a dep on having the prom chart installed (could be wrong), but that's probably ok

Reply:
Yes, it will create a dependency on Prometheus being available. I think it's reasonable since we should always have a monitoring stack deployed.

Reply:
yep, it is what it is :)